Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp

```diff
@@ -43,9 +43,6 @@ ControllerInterface g_controller_interface;
 //
 void ControllerInterface::Initialize(void* const hwnd)
 {
-	if (m_is_init)
-		return;
```

Comment:
Getting rid of just the one use of m_is_init is a little strange... we should either be checking it everywhere, or get rid of it.